ig/markdown/ExtensionsfrAccount.md

```diff
@@ -30,7 +30,8 @@
 ----
 
 
-**Name**: Extension Abrechnungsart ([Simplifier Projekt Link](https://simplifier.net/resolve?canonical=http://fhir.de/StructureDefinition/ExtensionAbrechnungsDiagnoseProzedur@1.4.0))
+**Name**: Extension Fallbezogene Abrechnungsrelevanz von Diagnosen und Prozeduren
+([Simplifier Projekt Link](https://simplifier.net/resolve?canonical=http://fhir.de/StructureDefinition/ExtensionAbrechnungsDiagnoseProzedur@1.4.0))
 
 **Beschreibung**: @``` from StructureDefinition where url = 'http://fhir.de/StructureDefinition/ExtensionAbrechnungsDiagnoseProzedur' select description```
```

Comment:
suggestion: Da hier keine "Beschreibung" steht: Entweder 1) eine Beschreibung hinzu in Abgrenzung zu dem "Hinweis" unten ? Oder 2) Lieber als "Profil" kennzeichnen.

Reply:
suggestion: analog dann oben für die andere Extension

Reply:
Wird durch Simplifier als leere Zeile herausgefiltert